WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -765,22 +769,45 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
         guard let post = post else {
             return
         }
+
         textView.isPrivate = post.isPrivate()
         textView.content = post.contentForDisplay()
```

Comment:
Unrelated to this PR, but I noticed it the other day, and this ☝️ is doing the same thing that we're already doing afterwards and generating the attributed string twice.

Reply:
So `textView.content = post.contentForDisplay()` is not necessary because the attributed text is set after it right?

Reply:
Yes, the content setter actually does:

```swift
attributedText = WPRichContentView.formattedAttributedStringForString(newValue)
```